pandas/tests/extension/arrow/test_bool.py

```diff
@@ -29,6 +29,10 @@ def data_missing():
     return ArrowBoolArray.from_scalars([None, True])
 
 
+def test_basic_equals(data):
+    assert pd.Series(data).equals(pd.Series(data))
```

Comment:
There is a `BaseMethodsTests` `test_equals` that could in theory be reused here. But since none of the other methods are implemented, we would need to skip all other tests, which would be a bit onerous. So fine with a custom test.